### PR TITLE
#595 Add new team members carousel

### LINKED ISSUE #595: Create a new carousel for the main page of The Hub
## Brief description:
The current carousel works but it can be remade to work better

## How to achieve it:
Research for library to make the carousel with and implement it

### PR DESCRIPTION
**Pull request by Pastedko**
Created a new carousel (#595)

Created a new carousel for the main page

resolves #595 

- [ ] Open the main page

### CODE REVIEW

File: packages/web/src/components/spa/MembersSection/test.css
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/test.css:1]: If you are not using this remove this file.

File: packages/web/src/components/spa/MembersSection/members.old.css
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/members.old.css:1]: Remove this

File: packages/web/src/components/spa/MembersSection/Carousel.jsx
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/Carousel.jsx:1]: Remove this

File: packages/web/src/components/spa/MembersSection/Carousel.old.old.jsx
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/Carousel.old.old.jsx:1]: Remove this

File: packages/web/src/components/spa/MembersSection/MembersCard.old.jsx
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/MembersCard.old.jsx:1]: Remove this

File: packages/web/src/components/spa/MembersSection/MembersSection.old.jsx
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/MembersSection.old.jsx:1]: Remove this

File: packages/web/src/components/spa/MembersSection/MobileCarousel.old.old.jsx
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/MobileCarousel.old.old.jsx:1]: Remove this

File: packages/web/src/components/spa/MembersSection/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        const slides = this.props.view;
+        const pictures = this.props.pictures;
+        this.state = {
+            sliders: pictures,
+            slides: slides,
+        };
+    }
+    sliders() {
+        return this.state.sliders.map(data => {
+            return (
```
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/Carousel.jsx:20]: Rename all tedko classNames. lol

File: packages/web/src/components/spa/MembersSection/MemberPage.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    const one = useMediaQuery({ query: '(max-width: 800px)' });
+    //const isFoldRes = useMediaQuery({ query: '(max-width: 350px)' });
+
+    const getMembers = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/members',
+        })
+            .then(res => {
+                // let localMembers = [];
```
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/MemberPage.jsx:28]: Remove comments

**Review by Aleks1908 — CHANGES_REQUESTED** [review]

File: packages/web/src/components/spa/MembersSection/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import './carousel.css';
+import { FaLinkedin } from 'react-icons/fa';
+
+export default class SimpleSlider extends Component {
+    constructor(props) {
```
Comment (Aleks1908) [inline packages/web/src/components/spa/MembersSection/Carousel.jsx:9]: Avoid using jquery. Look at how you have implemented it in the HackAUBG page and do it in the same way here

**Review by Aleks1908 — CHANGES_REQUESTED** [review]

**Comment by IvanObreshkov** [comment]
@Pastedko @Aleks1908 will this PR get closed :D

**Comment by asynchroza** [comment]
> @Pastedko @Aleks1908 will this PR get closed :D

They've set up a new React project. These changes are not required anymore.

**Pull request closed without merging**